src/scriptsys/script.h

```diff
@@ -3,10 +3,35 @@
 
 #define SCRIPT_DIR "script/"
 
+#define LUA_FN(n) { .name = #n, .func = func_##n }
+
+#define LUA_LIB(L, ...) luaL_Reg lib[] = \
```

Comment:
May want to wrap this in a do { ... } while(0) so that lib[] does not persist as a local variable

Reply:
Would it also be possible to just wrap it in a standard scope { } ?